docs/howto/openstack/nova/config-drive.md

```diff
@@ -0,0 +1,95 @@
+# Store metadata on a configuration drive
```

Comment:
Here we need an introductory paragraph that explains what problem this how-to solves.

Reply:
Done